controllers/progresses.js

```diff
@@ -48,6 +52,12 @@ const createProgress = async (req, res) => {
     body: { type },
   } = req;
   try {
+    const date = new Date();
+    if (date.getDay() === 0) {
+      return res.status(400).json({
+        message: PROGRESS_DOCUMENT_NON_WORKING_DAYS,
+      });
+    }
```

Comment:
can you try to handle it in validator. As validation checks are majorly handled in validator file.

Reply:
done, thanks for pointing it out